src/contrasts.jl

```diff
@@ -224,6 +266,17 @@ regression model, dummy coding leads to an intercept that is the mean of the
 dependent variable for base level.
 
 Also known as "treatment coding" (`contr.treatment` in R) or "one-hot encoding".
```

Comment:
> My feeling about point 2 there is that it could be very confusing to print a pseudo coefficient whose interpretation is qualitatively different from the other coefficients. I think it would be a good idea to provide some way to generate pseudo coefficients (e.g., full dummy coefficients) from the output (perhaps as an argument to coefs/coeftable). But I haven't thought carefully about how easy this is to do when there are multiple predictors.

AFAIK the interpretation of the omitted coefficient isn't really different from that of the actual coefficients when using effects coding: all of them measure the deviation from the mean associated with a level. OTC, viewing the base level as different from the others is weird, since one would get the same results after choosing a different base. And even for dummy coding, printing the name of the reference together with a `0` value is very useful (you can't interpret the coefficients without this information), especially since in some cases the reference can be `1` (e.g. when using odds ratios).

As regards the actual computation, I'm not sure the presence of multiple predictors would be an issue. Doesn't the omitted coefficient only depend on the contrast system? The tricky part is already handled when you choose whether to use full dummy coding or standard dummy coding.